### PR TITLE
Home page and login pages

### PR DESCRIPTION
**Pull request by veereshbv04**
Completed with all the E commerce pages.
Please review and suggest improvements

Live Link : [Link](https://bookstorecss.netlify.app/pages/home.html)

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+   
+}
+
+.subheading {
+    text-align: center;
+    max-width: 800px;
```
Comment (abhisheksahani-nova) [inline styles/index.css:80]: Use rem for pixel value more than 10px

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    margin: auto;
+    padding: 1rem;
+    text-transform: uppercase;
+}
+
+.offer-saving{
+    color: green;
```
Comment (abhisheksahani-nova) [inline styles/index.css]: Use css variables

File: styles/index.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+.center-div{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: auto;
+    padding: 6rem;
+}
+
+.green{
+    color: green;
```
Comment (abhisheksahani-nova) [inline styles/index.css]: use css variable everywhere, thats a good practice

**Review by abhisheksahani-nova — CHANGES_REQUESTED** [review]
Make the suggested changes , else everthing looks great

File: styles/index.css (hunk 2 above)
Comment (veereshbv04) [reply styles/index.css]: yeah will extract colors to root

File: styles/index.css (hunk 1 above)
Comment (veereshbv04) [reply styles/index.css:80]: I will change that now

File: styles/index.css (hunk 3 above)
Comment (veereshbv04) [reply styles/index.css]: yeah thanks, i will do that

**Review by PrashanthreddyAinala — COMMENTED** [review]
Be consistent while using colors, use variables in all places. Rest everything looks good for me.

File: styles/core.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+:root{
+    --primary-color:orange;
+    --background-color:rgb(187, 240, 207);
+    --text-color:blue;
+    --section-background:rgba(230, 240, 233, 0.767);
+    --dark-background:grey;
```
Comment (Prashantrajput11) [inline styles/core.css:6]: Be consistent with color value notations.

**Comment by veereshbv04** [comment]
> Be consistent while using colors, use variables in all places. Rest everything looks good for me.

Yeah I made chaanges and will push them now

**Pull request merged**